src/component/home.js

```diff
@@ -5,57 +5,91 @@ import {
   MdLabelOutline,
   MdOutlineArchive,
   MdDeleteOutline,
+  MdPushPin,
+  MdDelete
 } from "react-icons/md";
+import { useNotes } from "../context/noteProvider";
 
```

Comment:
You can use index  for shorter route path.

Reply:
Thanks . I will do it in refactor